modules/opensearch/opensearch.client.inc

```diff
@@ -516,6 +516,13 @@ function opensearch_execute(TingClientRequest $request) {
       if ($original_local_ids) {
         $request->setLocalIds($original_local_ids);
       }
+
+      // If response is empty array it means ther was no objects found for the
```

Comment:
typo: If response is empty array it means ther -> If response is an empty array, it means there

Reply:
PR is updated!